tests/scripts/validate_cluster.sh

```diff
@@ -20,7 +20,14 @@ set -xEe
 if [ -z "$DAEMON_TO_VALIDATE" ]; then
   DAEMON_TO_VALIDATE=all
 fi
-OSD_COUNT=$2
+# The second script arg is optional and depends on the daemon
+if [ "$DAEMON_TO_VALIDATE" == "rgw" ]; then
+  export OBJECT_STORE_NAME=$2
+else
+  export OSD_COUNT=$2
+  # default to the name of the object store from object-a.yaml
+  export OBJECT_STORE_NAME=store-a
```

Comment:
I don't think these need to be `exported`, unless I missed something. 

IIRC, vars exported in a script get set in the calling terminal, and then they can propagate to future script calls. I usually find it best not to export vars unless it's needed.

Reply:
Initially I wasn't exporting the vars and the variable was being evaluated as empty inside the `timeout` block where this var was needed. Adding the `export` resolved the issue. Perhaps there is a different syntax needed for evaluation in the `timeout` block below? Instead of investigating the alternative, the `export` seemed harmless enough since it's just for testing.